libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/navigation/metrics/MetricEvent.kt

```diff
@@ -20,7 +18,21 @@ interface MetricEvent {
     annotation class Metric
 
     @Metric
-    val metric: String
+    val metricName: String
```

Comment:
Have those changes been released already? If yes, maybe we should avoid this breaking change.

Reply:
As I understand, this changes for 1.0 so it is not released.... but yeah, I'm not sure 100%, because I confused a bit with our pre-1.0 versions) @Guardiola31337 Could you please clarify this point?

Reply:
Well, it is in the `navigation/v5` package, so now it'd be released with the legacy code.

Reply:
I think we're good here. These changes only were included as part of the `0.43.0-alpha.1` release (Kotlin conversion of the current SDK).

As of the legacy code / SDK we're patching bug fixes in this branch [`base-v0.42.1`](https://github.com/mapbox/mapbox-navigation-android/commits/base-v0.42.1) and cutting the releases from there.

Reply:
Sounds good :+1: